handbook/engineering/campaigns/index.md

```diff
@@ -82,6 +70,18 @@ Our users use more than just their code host and Sourcegraph. They use ticket tr
 
 We want to discover what external systems our users want to use campaigns with, and ensure that we can integrate with them. For example, an organization that uses JIRA will likely want to be able to link tickets to campaigns and have the state updated as the campaign is executed.
 
+## Roadmap
+
+Our team's public roadmap is part of our larger [Product roadmap](https://about.sourcegraph.com/handbook/product/roadmap#campaigns).
+
+### Roadmap Process
+
+Our roadmapping process is a team effort. As we receive customer feedback, and as we come up with cool new features, we record these in one of three places. Customer feedback goes into our [team Productboard page](https://sourcegraph.productboard.com/feature-board/2104383-campaigns). Well-defined tasks live in our [team backlog](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aopen+is%3Aissue+label%3Ateam%2Fcampaigns+milestone%3ABacklog). And larger, less defined ideas live in the bottom section of our [private roadmap doc](https://docs.google.com/document/d/1zRTfK6mENKicfLwDaWgLk1dBvQVKDg-J7pwjGg8tpps/edit#), below the _fog of war_ line.
```

Comment:
Side-note (not a blocker): does the private roadmap doc need to be private? I don't think there's sensitive information in there and potential candidates often enjoy looking at the roadmap. I'd rather have us "default to public" than "default to private and maybe make it public"

Reply:
My default is "private when it relates to customer data" which is something that was very much drilled into me from my last job (to the point where we couldn't even say "$COMPANY is a customer of ours"). 😄 That roadmap doc has information about feature requests from several customers. So until I hear otherwise, I'm going to stick with that default.

Otherwise, I totally agree with you, and would have no problem making it public.

Reply:
More context:

Things we don't want to be public include customer names (I see some in that doc), company financials or sensitive metrics (e.g. revenue, I don't see any such things).

Even if customer X is public (i.e. they are on our homepage), we would not want to attribute specific feedback/quotes to that customer (since they might not have given permission to us to share that quote).

We are open, but we assume that our customers are not open.

Reply:
Got it! Thanks for both answers :)

Reply:
> We are open, but we assume that our customers are not open.

Great way to put that, thanks!